source/libs/page-detect.ts

```diff
@@ -366,6 +367,11 @@ export const _isTrending = [
 	'https://github.com/trending/unknown'
 ];
 
+export const isBranches = (): boolean => /^branches/.test(getRepoPath()!);
```

Comment:
It should be:
```suggestion
export const isBranches = (): boolean => getRepoPath()?.startsWith('branches') ?? false;
```

Even though it's more verbose, it's IMHO more readable.